plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzers/base_analyzer.py

```diff
@@ -194,11 +205,14 @@ def __init__(
     def _remove_empty_lines_from_csv(input_file):
         temp_file = input_file + ".temp"
 
-        with open(
-            input_file, "r", newline="", encoding=DataConstants.UTF8ENCODING
-        ) as infile, open(
-            temp_file, "w", newline="", encoding=DataConstants.UTF8ENCODING
-        ) as outfile:
+        with (
+            open(
+                input_file, "r", newline="", encoding=DataConstants.UTF8ENCODING
+            ) as infile,
+            open(
+                temp_file, "w", newline="", encoding=DataConstants.UTF8ENCODING
+            ) as outfile,
+        ):
```

Comment:
Why did you change it? I'm not sure I see the benefit

Reply:
ruff requires me to change

Reply:
I tried to fix it, really strange.... lets keep this change.
I pushed some fixes to the ruff step